diff --git a/nimble/host/include/host/ble_ll_conn_params.h b/nimble/host/include/host/ble_ll_conn_params.h
--- a/nimble/host/include/host/ble_ll_conn_params.h
+++ b/nimble/host/include/host/ble_ll_conn_params.h
@@ -33,7 +33,10 @@
 #ifndef BLE_LL_CONN_PARAMS_H_
 #define BLE_LL_CONN_PARAMS_H_
 
+/* FUZZING_STATE: Connection parameters */
 extern uint16_t g_ble_ll_conn_params_upd_timeout;
+extern bool g_ble_ll_conn_fuzz_mode;
 
 int ble_ll_conn_params_update(struct ble_hs_conn *conn);
+void ble_ll_conn_params_set_limits(uint16_t min_int, uint16_t max_int);
 
 #endif
 