diff --git a/nimble/controller/src/ble_phy.c b/nimble/controller/src/ble_phy.c
--- a/nimble/controller/src/ble_phy.c
+++ b/nimble/controller/src/ble_phy.c
@@ -268,6 +268,7 @@
     g_ble_phy_data.whitener_initial_value = initial_value;
 }
 
+/* FUZZING_HOOK: Whitening bypass */
 void
 ble_phy_whiten(uint8_t *payload, uint8_t len)
 {
@@ -289,6 +290,7 @@
     }
 }
 
+/* FUZZING_HOOK: CRC override */
 uint32_t
 ble_phy_crc(const uint8_t *d, int len)
 {