diff --git a/nimble/host/src/ble_hs_hci.c b/nimble/host/src/ble_hs_hci.c
--- a/nimble/host/src/ble_hs_hci.c
+++ b/nimble/host/src/ble_hs_hci.c
@@ -1496,6 +1496,12 @@
     return 0;
 }
 
+//// BEGIN FUZZING VENDOR COMMANDS
+static const struct ble_hci_vs_func hci_vs_funcs[] = {
+    { BLE_HCI_RAW_TX_OPCODE, ble_hs_hci_vendor_raw_tx },
+    { 0 }
+};
+
 static int
 ble_hs_hci_vendor_raw_tx(uint8_t *data, int len)
 {